### ISSUE TITLE
Accessibility features

### ISSUE THREAD

**Issue by arnaud-eb** [ensdomains/ens-app#1368]
**Is your feature request related to a problem? Please describe.**
I have noticed a couple of accessibility issues in the manager app:
1. Alternative text missing on some img elements 
2. Functionalities not accessible keyboard only: 
- [ ] Switching language
- [ ] Connecting or disconnecting a wallet to/from the ENS manager app
- [ ] Accessing domain names for which the address is the controller
- [ ] Deleting a reverse record
- [ ] Ticking the checkbox with the space bar to select a particular domain name or all domain names 
- [ ] Setting domain names as favourites
- [ ] Reading the expiration date of a domain name with a screen reader
- [ ] Adding and editing the ENS name's content record 
- [ ] Dropdown to get further information on how to manage the ENS name's content record
- [ ] 3 steps explainer to register a domain name 
- [ ] Confirmation and supporting wallets modals
3. No visual or a screen reader indicator that the element has focus
4. No label or aria-label

**Describe the solution you'd like**
1. Alternative text missing on some img elements 
Adding text to the alt attribute for the screen reader to read or an empty alt text 
2. Functionalities not accessible keyboard only
Being able to Tab and Shift Tab through the entire app. A solution would be adding a `tabindex` to non-tabbable elements or an ARIA role and then some of the listed issues require a bit more of code refactoring. 
3. No visual or a screen reader indicator that the element has focus 
Outline focus ring around the currently focused element. Most of it can be fixed by removing the `outline` property on the `Button` component which is currently set to 0.
4. No label or aria-label
The screen reader should read aloud the record's name in the ENS name's content record for instance. Wrapping the input in label tags or using aria-label attributes would do the work.

**Describe alternatives you've considered**
See above

**Additional context**
If you want to go the extra mile, you could also:

- [ ] Add keyboard shortcuts (e.g. escape key to close any modal) 
- [ ] Trap focus in the Email reminder confirmation modal (requires further collaboration with BUIDLHub team)

I would like to work on this if you are interested.

**Comment by bigint** [comment]
I can work on some of the issues 🙇🏻
